drivers/backup/backup.go

```diff
@@ -16,6 +16,14 @@ type Image struct {
 	Version string
 }
 
+type RuleStruct struct {
```

Comment:
Lets not have a variable name with `Struct` in it. It should only be called `Rule`

Reply:
sure

Reply:
done